CourseApp/UserInterface.cs

```diff
@@ -0,0 +1,280 @@
+1 - Create another custom file
+2 - Back to previous page
+3 - Exit program
+What you want? Enter integer values.");
+            switch (input.InputInt(1, 2))
+            {
+                case 1:
+                    CustomFile(customFile);
+                    break;
+
+                case 2:
+                    CustomFilePage(customFile);
+                    break;
+
+                case 3:
+                    Environment.Exit(0);
+                    break;
+            }
+        }
+
+        private void GetCustomFiles(List<string> customFile)
+        {
+            var input = new InputValues();
+            Console.Clear();
+            if (IsFileCreated)
+            {
+                Console.WriteLine($"Your file{((customFile.Count == 1) ? ":" : "s:")}");
+                for (int i = 0; i < customFile.Count; i++)
+                {
+                    Console.WriteLine($"{i + 1} - {customFile[i]}");
+                }
+            }
+            else
+            {
+                Console.WriteLine("You dont create any files");
+            }
+
+            Console.WriteLine(@"
+Available actions:
+1 - Create custom file
+2 - Back to previous page
+3 - Exit program
+What you want? Enter integer values.");
+            switch (input.InputInt(1, 3))
+            {
+                case 1:
+                    CustomFile(customFile);
+                    break;
+
+                case 2:
+                    CustomFilePage(customFile);
+                    break;
+
+                case 3:
+                    Environment.Exit(0);
+                    break;
+            }
+        }
+
+        private string EnumValueName()
+        {
+            var input = new InputValues();
+            Console.Clear();
+            Console.WriteLine($"Choose name. Enter integer values.");
+            for (int i = 0; i < Enum.GetNames(typeof(Name)).Length; i++)
+            {
+                Console.WriteLine($"{i + 1} - {Enum.GetName(typeof(Name), i)}");
+            }
+
+            return Enum.GetName(typeof(Name), input.InputInt(1, Enum.GetNames(typeof(Name)).Length) - 1);
+        }
+
+        private string EnumValueExtension()
+        {
+            var input = new InputValues();
+            Console.Clear();
+            Console.WriteLine($"Choose extension. Enter integer values.");
+            for (int i = 0; i < Enum.GetNames(typeof(Extension)).Length; i++)
+            {
+                Console.WriteLine($"{i + 1} - {Enum.GetName(typeof(Extension), i)}");
+            }
+
+            return Enum.GetName(typeof(Extension), input.InputInt(1, Enum.GetNames(typeof(Extension)).Length) - 1);
+        }
+
+        private string EnumValueWeightModificator()
+        {
+            var input = new InputValues();
+            Console.Clear();
+            Console.WriteLine($"Choose weightModificator. Enter integer values.");
+            for (int i = 0; i < Enum.GetNames(typeof(WeightModificator)).Length; i++)
```

Comment:
I guess it should be seperate variable and then we can get element by index (e.g. names[i])